wp-job-manager-template.php

```diff
@@ -1134,7 +1134,7 @@ function get_the_company_tagline( $post = null ) {
 function the_company_twitter( $before = '', $after = '', $echo = true, $post = null ) {
 	$company_twitter = get_the_company_twitter( $post );
 
-	if ( 0 === strlen( $company_twitter ) ) {
+	if ( ! $company_twitter || 0 === strlen( $company_twitter ) ) {
```

Comment:
I think we can simplify this to:
```suggestion
	if ( empty( $company_twitter ) ) {
```